app/experimenter/settings.py

```diff
@@ -405,5 +405,6 @@
 DEFAULT_FILE_STORAGE = "storages.backends.gcloud.GoogleCloudStorage"
 GS_PROJECT_ID = "experiments-analysis"
 GS_BUCKET_NAME = "mozanalysis"
+GS_LOCATION = "statistics"
```

Comment:
I'm not very familiar with this code base, but this should be the only changed required based on the docs: https://django-storages.readthedocs.io/en/latest/backends/gcloud.html